.test.mk

```diff
@@ -275,10 +275,10 @@ test-coverity: build-coverity
 # LuaJIT integration testing #
 ##############################
 
-test-luajit-linux-x86_64: build run-luajit-test run-test
+test-luajit-Linux-x86_64: build run-luajit-test run-test
```

Comment:
Actually, I like this naming :) Can we keep it further?

Reply:
We use this values within LuaJIT matrix and I, actually, don't want to make additional tweaks to obtain the values for the current naming.

I suggest to try the new naming and return to this question, if we face issues with it (either Makefile targets mislookup or its complex maintenance).